examples/nested_client.py

```diff
@@ -0,0 +1,30 @@
+"""This example shows how to nest Client in task. Please read graphtask_nested_client.py for more information."""
+
+from scaler import Client
+from scaler.client.client import Client
+from scaler.cluster.combo import SchedulerClusterCombo
+
+
+# Calculate fibonacci sequence with nested client.
+# Each intermediate call in the recursive process is submitted to the client.
+def fibonacci(clnt: Client, n: int):
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        a = clnt.submit(fibonacci, clnt, n - 1)
+        b = clnt.submit(fibonacci, clnt, n - 2)
+        return a.result() + b.result()
```

Comment:
Imo, no reason to shorten the parameter name here.

```suggestion
def fibonacci(client: Client, n: int):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        a = clnt.submit(fibonacci, client, n - 1)
        b = clnt.submit(fibonacci, client, n - 2)
        return a.result() + b.result()
```